Sources/FeaturevisorSDK/Instance.swift

```diff
@@ -413,73 +414,61 @@ public class FeaturevisorInstance {
           }
       }
 
-    //  func getVariableBoolean(
-    //    feature: Feature,
-    //    variableKey: String,
-    //    context: Context
-    //  ) -> Bool? {
-    //      return getVariableBoolean(featureKey: feature.key, variableKey: variableKey, context: context)
-    //  }
-    //  func getVariableBoolean(
-    //    featureKey: FeatureKey,
-    //    variableKey: String,
-    //    context: Context
-    //  ) -> Bool? {
-    //      let variableValue = self.getVariable(featureKey: featureKey, variableKey: variableKey, context: context)
-    //      // TODO: implement in Swift
-    //    //return getValueByType(variableValue, "boolean") as boolean | undefined;
-    //  }
+      func getVariableBoolean(
+        feature: Feature,
+        variableKey: VariableKey,
+        context: Context) -> Bool? {
+            return getVariableBoolean(featureKey: feature.key, variableKey: variableKey, context: context)
+      }
+    
+      func getVariableBoolean(
```

Comment:
any reason why the method is repeating? I see `getVariableBoolean()` appear twice now.

same for `getVariableString()` few lines below.

Reply:
I was uncommenting the code where we had these two methods with one difference in parameters - `featureKey` vs `feature`. If we don't want to (I didn't check the JS SDK) then I can remove on of them here and the similar below ones.

Reply:
ahhh I notice it now! `feature` and `featureKey`.

given this is public API, my expectation is people will always pass the `featureKey`. 

the `feature` object is only possible to use internally in the SDK.

keeping that in mind, I would request we keep only `featureKey` similar to how it's done in JS SDK.

JS API: 

- Docs: https://featurevisor.com/docs/sdks/#type-specific-methods
- Code: https://github.com/fahad19/featurevisor/blob/8bb0ab21bca434ff9c02029894795ea09541b7aa/packages/sdk/src/instance.ts#L1176

Reply:
Pushed the update. The test results are the same ;)